src/com/github/Kraken3/AFKPGC/BotDetector.java

```diff
@@ -148,7 +145,6 @@ public synchronized void doDetector() {
 						if (la.loggedLocations.size() == maxLocations) {
 							int itWasntMeISwear = la.calculateMovementRadius();
 							if (itWasntMeISwear < smallestMovedDistance) {
-								smallestMovedDistance = itWasntMeISwear;

```

Comment:
That's just a leftover from when I was looking for the player who moved the least. With this, a player at the beginning of the list who didnt move much could prevent anyone else from being suspected, simply because he sets the limit very low.